diff --git a/README.mkdn b/README.mkdn
index f14c344..ce3f0f7 100644
--- a/README.mkdn
+++ b/README.mkdn
@@ -1362,6 +1362,12 @@ information to the console. Check that Chrome starts:
 
 `chrome`
 
+On MacOS, the executable can usually be be found inside the `Google Chrome`
+package in the `Applications` directory and so can be tested with
+something like the following:
+
+`/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version`
+
 ## Chrome versions
 
 Note that the Chrome version numbers do not denote availability of features.
diff --git a/lib/WWW/Mechanize/Chrome.pm b/lib/WWW/Mechanize/Chrome.pm
index f6e39f8..867238b 100644
--- a/lib/WWW/Mechanize/Chrome.pm
+++ b/lib/WWW/Mechanize/Chrome.pm
@@ -4618,6 +4618,12 @@ information to the console. Check that Chrome starts:
 
 C<< chrome >>
 
+On MacOS, the executable can usually be be found inside the `Google Chrome`
+package in the `Applications` directory and so can be tested with
+something like the following:
+
+`/Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --version`
+
 =head2 Chrome versions
 
 Note that the Chrome version numbers do not denote availability of features.
